dashboards/grafana-dashboard-subscription-watch.configmap.yaml

```diff
@@ -3220,9 +3342,9 @@ data:
           },
           {
             "current": {
-              "selected": false,
-              "text": "AWS insights-stage",
-              "value": "AWS insights-stage"
+              "selected": true,
+              "text": "AWS insights-prod",
```

Comment:
`stage` vs `prod`

Reply:
Not sure I follow, This would change the default selection to be production instead of stage. Is the request here to not do that or will it break something?